content/root/best-practice-guide.textile

```diff
@@ -195,9 +184,48 @@ h3(#embedding-client-id-in-token). Embedding client ID in the token to ensure tr
 
 A client ID serves as an identification for your clients when they are present on a channel. Your clients can set their own ID using the @clientId@ attribute, but in that case, there's a possibility that some of your clients can pretend to be someone else. To prevent this, you can embed a @clientId@ in the token issued to your clients from your auth server. Doing so ensures that the @clientId@ is set by your auth server, thus eliminating the chance of the end clients emulating as others.
 
+h3(#presence-volume). Understanding the number of presence messages that can be produced
+
+It is important when using the "presence":/realtime/presence feature to understand the number of messages it can produce. If you have subscribers and presence members present on a channel then the number of messages used can increase rapidly. As an example, consider 200 clients subscribed to presence events on a channel and all of them join and leave the presence set within a few minutes. This would result in the following messages:
+
+* 200 presence messages published for the enter event.
+* 200 x 200 (40,000) messages subscribed to for the enter events.
+* 200 presence messages published for the leave event.
+* 200 x 200 (40,000) presence messages subscribed to for the leave event.
+
+This highlights the potential for 80,400 messages to be sent in a very short space of time on a single channel. As additional clients subscribe and enter the presence set, the number of messages quickly multiplies.
+
+In some instances this is the behavior required of your application. In other scenarios you may prefer to "only have a server subscribed to presence events":#being-present-without-subscribing-to-presence and the clients are only present on the channel.
+
+h3(#being-present-without-subscribing-to-presence). Being present without subscribing to presence events
+
+It is possible to have clients be present on a channel without them receiving "presence change events":/realtime/presence#presence-states. Efficient use of "capabilities":/core-features/authentication#capabilities-explained can prevent all clients from receiving presence notifications while still allowing them to enter the presence set. This is a common scenario when you just want to know how many people are present on a channel at any point in time, for example by only having a server subscribed to the presence events. 
+
+An example of achieving this would be to use one channel for generic communications and another for the presence set. The following capabilities demonstrate this for clients and for servers:
+
+For clients:
+
+```[json]
+{
+  "chatPresence": ["presence"],
+  "chat": ["publish", "history", "subscribe"],
+}
+```
+
+For servers:
+
+```[json]
+{
+  "chatPresence": ["presence", "subscribe"],
+  "chat": ["publish", "history", "subscribe"],
+}
+```
+
+You could then write a mechanism within your application to periodically relay the "presence set":#use-existing-presence-set from the server to clients.
```

Comment:
> You could then write a mechanism within your application to periodically relay the "presence set":#use-existing-presence-set from the server to clients.

I'm not sure this makes much sense? This paragraph has been about a pattern you can use if your clients _don't_ need to know who's present, they just need to be able to enter presence. Then you're ending it by saying that if the clients need to know the presence set they can be relayed it by the customer's own servers. But if the clients do need to know the presence set they should just subscribe to it

Reply:
Very good point... I've removed that sentence completely.